aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -64,6 +64,7 @@ systemd:
           --volume var-log,kind=host,source=/var/log \
           --mount volume=var-log,target=/var/log \
           --insecure-options=image"
+        Environment=KUBELET_CGROUP_DRIVER=${cgroup_driver}
```

Comment:
Why is this needed to upgrade kubernetes version?

Reply:
This is not needed since we have a better way of dealing with this in an automated way #62.